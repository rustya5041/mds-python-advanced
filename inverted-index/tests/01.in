Anarchism
Anarchism is
is Anarchism runtime-flexible
in-prison
in-prison slows.
Enough:
Enough: in-prison slows.
overlapped Alvy
independence serves strength
independence serves strength as4d5f6g7hj
Press Stephen capacity care
history, America, female records
Center exception individuals suggests
reduce agreed asked begin summer technology Like River
Like River summer
Research Russian deal
Research Russian deal 506
requires purpose School
