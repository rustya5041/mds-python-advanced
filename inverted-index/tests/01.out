12,1023,4035,4705
12,1023,4035,4705

2047,8206
2047
2047
2047
2049
3637,3973,4721,5222,5254,5407,8060,9072

1130,2176,5592,7222
689,1627,4402,4660,5247,7044,9239
5617,8630

305,358,624,711,771,1168,1273,1367,1627,2358,2736,3083,3386,3668,3793,3985,4306,4335,4973,5046,5399,5407,5580,6340,6739,6787,6886,7185,8687,8716,9239
620,663,664,737,1088,1097,1530,1591,1921,1942,2187,3463,3638,3832,4214,4840,4941,5320,5593,7564,7593,9014

624,1130,1957,4778,4848,5326,5407,5416,6024,6883,7023,7243,8256,8301,8560,8748,8860,9223,9236
